docs/development/new-package-manager-template.md

```diff
@@ -25,14 +25,14 @@ Explain how this package manager is different from existing ones.
 
 ### What kind of package files, and names, does this package manager use?
 
-### Which [`fileMatch`](../usage/configuration-options.md#filematch) pattern(s) should Renovate use?
+### Which [`filePatterns`](../usage/configuration-options.md#filepatterns) pattern(s) should Renovate use?
```

Comment:
We decided on Slack the config option shoul be called `managerFilePatterns`. I'm commenting here to help us remember that change. 😉 

Todos:

- [ ] Update the PR title on GitHub
- [ ] Refactor from `filePatterns` to `managerFilePatterns`
- [ ] Update "migration code" so that Renovate can create a config migration PR to migrate users from the old term to the new term?